src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -1981,46 +1942,28 @@ std::vector<uint64_t> BlockchainLMDB::get_block_cumulative_rct_outputs(const std
 
   if (heights.empty())
     return {};
-  for (size_t i = 1; i < heights.size(); ++i)
-    if (heights[i] < heights[i-1])
-      throw0(DB_ERROR("Invalid heights array"));
   res.reserve(heights.size());
 
   TXN_PREFIX_RDONLY();
-  RCURSOR(rct_distribution);
+  RCURSOR(block_info);
 
   MDB_stat db_stats;
   if ((result = mdb_stat(m_txn, m_blocks, &db_stats)))
     throw0(DB_ERROR(lmdb_error("Failed to query m_blocks: ", result).c_str()));
-  if (heights.back() >= db_stats.ms_entries)
-    throw0(BLOCK_DNE(std::string("Attempt to get rct distribution from height " + std::to_string(heights.back()) + " failed -- block size not in db").c_str()));
+  for (size_t i = 1; i < heights.size(); ++i)
```

Comment:
Why start from 1 instead of 0?

Reply:
Because it used to test whether [n] was >= [n-1] :) Fixed.